elements/lisk-p2p/src/peer_directory/tried_list.ts

```diff
@@ -111,7 +110,6 @@ export class TriedList extends PeerList {
 		};
 
 		bucket.set(incomingPeerId, updatedTriedPeerInfo);
```

Comment:
Not directly related to this PR but you can simply do `bucket.set(incomingPeerId, { ...foundPeer, numOfConnectionFailures: numOfConnectionFailures + 1})`. This way you don't need all references to `peerInfo`, `dateAdded` and `updatedPeerInfo` above